buildscripts/kokoro/android-apk-stats.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -exu -o pipefail
+
+export GRADLE_OPTS=-Xmx512m
+export PROTOBUF_VERSION=3.5.1
+export LDFLAGS=-L/tmp/protobuf/lib
+export CXXFLAGS=-I/tmp/protobuf/include
+export LD_LIBRARY_PATH=/tmp/protobuf/lib
+export OS_NAME=$(uname)
+
+cd ./github/grpc-java
+
+# Proto deps
+buildscripts/make_dependencies.sh
+ln -s "/tmp/protobuf-${PROTOBUF_VERSION}/$(uname -s)-$(uname -p)" /tmp/protobuf
+
+./gradlew install
+cd examples/android/helloworld
+./gradlew build
+IFS=, read -r new_bytes new_kb new_mb < <(sed -n 2p app/build/outputs/apksize/release/release.csv)
+IFS=, read -r new_methods new_fields new_classes < <(sed -n 2p app/build/outputs/dexcount/release.csv)
+
+cd ../../../
+git checkout master
+./gradlew install
+cd examples/android/helloworld/
+./gradlew build
+IFS=, read -r old_bytes old_kb old_mb < <(sed -n 2p app/build/outputs/apksize/release/release.csv)
+IFS=, read -r old_methods old_fields old_classes < <(sed -n 2p app/build/outputs/dexcount/release.csv)
+
+bytes_delta=$((new_bytes-old_bytes))
+methods_delta=$((new_methods-old_methods))
+fields_delta=$((new_fields-old_fields))
+
+# Kororo URLs are in the form "grpc/job/macos/job/master/job/grpc_build_artifacts"
+KOKORO_JOB_PATH=$(echo "${KOKORO_JOB_NAME}" | sed "s|/|/job/|g")
+
+cd ../../../
+mkdir -p reports
+echo '<html><head></head><body>' > reports/kokoro_index.html
+echo '<h1>'${KOKORO_JOB_NAME}', build '#${KOKORO_BUILD_NUMBER}'</h1>' >> reports/kokoro_index.html
+echo '<h2><a href="https://kokoro2.corp.google.com/job/'${KOKORO_JOB_PATH}'/'${KOKORO_BUILD_NUMBER}'/">Kokoro build dashboard (internal only)</a></h2>' >> reports/kokoro_index.html
```

Comment:
Just a heads up: we might be moved from kokoro2 in the future. Just saying so that if this link stops working, you'll know the reason this link broke the last time for us last time.